status.go

```diff
@@ -0,0 +1,50 @@
+package migrate
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/johngibb/migrate/db"
+	"github.com/johngibb/migrate/source"
+)
+
+// Status displays every migration, and whether it's been applied yet.
+func Status(src *source.Source, db *db.Client) error {
+	migrations, err := src.FindMigrations()
+	if err != nil {
+		return err
+	}
+	applied, err := db.GetMigrations()
+	if err != nil {
+		return err
+	}
+	isApplied := func(name string) bool {
+		for _, a := range applied {
+			if a.Name == name {
+				return true
+			}
+		}
+		return false
+	}
+
+	w := maxNameWidth(migrations)
+	for _, m := range migrations {
+		status := "pending"
+		if isApplied(m.Name) {
+			status = "applied"
+		}
+
+		log.Printf("%-"+strconv.Itoa(w)+"s  %s\n", m.Name, status)
```

Comment:
stylistic: double space? guessing it’s to accommodate what you do with `maxNameWidth`.

Reply:
I just liked how it looked. What would you suggest instead?

Reply:
a single space

Reply:
Fair enough, I'll update. Thanks for the feedback my esteemed colleague!